### PR TITLE
Updated the setup tool for the HTML backend

### PR DESCRIPTION
**Pull request by MonsterOfCookie**
Updated the setup tool to handle the deprecation of the Jetty plugin from the Gradle distribution (4+)
Moved over to using a simple file server to serve the content whilst the SuperDev mode is handled by GWT and it's internal Jetty
Changed the GWT plugin to a newer one, it's fork of what was there before but this is actively maintained.

This is what I use for my HTML5 based projects now - Thought I should give back to the community :)

### CODE REVIEW

File: extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (Darkyenus) [inline extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/gradle/wrapper/gradle-wrapper.properties]: This seems to update Gradle to 4.1 - there has been some discussion about that: #4698 (and maybe more)

Comment (MonsterOfCookie) [inline extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/gradle/wrapper/gradle-wrapper.properties]: Ah good point. This was just more out of habit from my projects.

File: extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/html/GdxDefinitionSuperdev [hunk 2]
```diff
@@ -1,12 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE module PUBLIC "-//Google Inc.//DTD Google Web Toolkit trunk//EN" "http://google-web-toolkit.googlecode.com/svn/trunk/distro-source/core/src/gwt-module.dtd">
 <module rename-to="html">
-%GWT_INHERITS%
```
Comment (Tom-Ski) [inline extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/html/GdxDefinitionSuperdev]: Why is this removed?

Comment (MonsterOfCookie) [inline extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/html/GdxDefinitionSuperdev]: Wasn't needed as it is inherited from the GdxDefinition.gwt.xml

If it is needed, I haven't found the use case where it breaks without it :/

**Comment by Tom-Ski** [comment]
Looks good, will have a play with this implementation when I upgrade to 4 for the setup tool. I have updated to the replacement gretty plugin for the main libgdx repo, but I'm not entirely happy with it. This looks like a nicer option.

**Comment by MonsterOfCookie** [comment]
Yea, when I took everything apart and realised Jetty was just being used to serve static content, I just thought why not KISS it.

**Comment by Tom-Ski** [comment]
I agree, we'll use this when we upgrade. Just waiting on https://youtrack.jetbrains.com/issue/IDEA-122904 ....

**Comment by MrStahlfelge** [comment]
Looks like this is already used now without a merge - can be closed now.
